deployment/GA02HYPM_Deploy.csv

```diff
@@ -1,43 +1,43 @@
+AT26-30,,NBP1510,,GA02HYPM-WFP03-05-VEL3DL000,1,1,2015-03-16T19:02:00,2015-11-15T11:38:00,CGMGA-02HYPM-00001,CGVEH-GWFPGX-10406,CGINS-VEL3DL-01117,-42.98167,-42.4985,,5116,5200,
+#NBP1510,,AT39-03,,GA02HYPM-MPM01-02-ZPLSGA009,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,,,,,,,5196,not deployed
+#NBP1510,,AT39-03,,GA02HYPM-MPM01-02-ZPLSGA010,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,,,,,,,5196,not deployed
+NBP1510,,AT39-03,,GA02HYPM-RIM01-00-SIOENG000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,,CGCON-SIOCON-14880,-42.98083,-42.54192,,5025,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-RIM01-02-CTDMOG039,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,,CGINS-CTDMOG-13606,-42.98083,-42.54192,,163,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-WFP02-00-WFPENG000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,CGVEH-GWFPGX-45203,CGCON-GWFPGX-45203,-42.98083,-42.54192,,2573,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-WFP02-01-FLORDL000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,CGVEH-GWFPGX-45203,CGINS-FLORDL-04015,-42.98083,-42.54192,,2573,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-WFP02-03-DOSTAL000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,CGVEH-GWFPGX-45203,CGINS-DOSTAL-02197,-42.98083,-42.54192,,2573,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-WFP02-04-CTDPFL000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,CGVEH-GWFPGX-45203,CGINS-CTDPFL-00152,-42.98083,-42.54192,,2573,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-WFP02-05-VEL3DL000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,CGVEH-GWFPGX-45203,CGINS-VEL3DL-01255,-42.98083,-42.54192,,2573,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-WFP03-00-WFPENG000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,CGVEH-GWFPGX-45204,CGCON-GWFPGX-45204,-42.98083,-42.54192,,5015,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-WFP03-01-FLORDL000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,CGVEH-GWFPGX-45204,CGINS-FLORDL-04016,-42.98083,-42.54192,,5015,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-WFP03-03-DOSTAL000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,CGVEH-GWFPGX-45204,CGINS-DOSTAL-02196,-42.98083,-42.54192,,5015,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-WFP03-04-CTDPFL000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,CGVEH-GWFPGX-45204,CGINS-CTDPFL-00153,-42.98083,-42.54192,,5015,5196,fell to seafloor approx 2016-06-01
+NBP1510,,AT39-03,,GA02HYPM-WFP03-05-VEL3DL000,2,1,2015-11-16T18:36:00,2018-01-15T09:30:00,CGMGA-02HYPM-00002,CGVEH-GWFPGX-45204,CGINS-VEL3DL-01254,-42.98083,-42.54192,,5015,5196,fell to seafloor approx 2016-06-01
```

Comment:
For Deployment 2, we should use 2016-06-01 as the stopDateTime as after this point, the data is not valid.  We can put in the note "Physically recovered on 2018-01-15".  All else looks good.